### PR TITLE
Add edit existing education functionality

### PR DESCRIPTION
**Pull request by david-wagih**

### CODE REVIEW

**Comment by fissoreg** [comment]
Thank you @david-wagih for the PR! :)
I would expect the test for `Education` to pass, but it seems like that's not the case. Am I missing something here?

File: app.py [hunk 1]
```diff
@@ -100,3 +104,56 @@ def add_skill():
     data["skill"].append(new_skill)
 
     return jsonify({"id": data["skill"].index(new_skill)})
+
+
+def add_experience():
+    """
+    Add an experience using POST method
```
Comment (fissoreg) [inline app.py:111]: It doesn't look like this function is specific to POST, but rather the fact that we are dealing with a POST request depends on the code above. If that's the case, then the function is making assumption about the context in which it is called, which is in general not desirable (functions are supposed to be atomic and independent on external variables/context). I would change this comment.

**Review by fissoreg — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (david-wagih) [reply app.py:111]: done

File: app.py [hunk 2]
```diff
@@ -100,3 +104,56 @@ def add_skill():
...
+        req["logo"],
+    )
+    data["experience"].append(new_experience)
+
+    return jsonify({"id": data["experience"].index(new_experience)})
+
+
+def edit_education():
+    """
+    Edit an education using PUT method.
```
[the remark this answers is not in the document]
Comment (david-wagih) [reply app.py:138]: done

**Comment by david-wagih** [comment]
@fissoreg can you review this one

**Pull request merged**